specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-07-02-preview/managedClusters.json

```diff
@@ -6853,6 +6939,75 @@
       ],
       "description": "The OS option profile."
     },
+    "GuardrailsAvailableVersionsProperties": {
+      "type": "object",
+      "description": "Whether the version is default or not and support info.",
+      "properties": {
+        "isDefaultVersion": {
+          "type": "boolean",
+          "readOnly": true
+        },
+        "support": {
+          "type": "string",
+          "enum": [
+            "Preview",
+            "Stable"
+          ],
+          "x-ms-enum": {
+            "name": "GuardrailsSupport",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Preview",
+                "description": "The version is preview. It is not recommended to use preview versions on critical production clusters. The preview version may not support all use-cases."
+              },
+              {
+                "value": "Stable",
+                "description": "The version is stable and can be used on critical production clusters."
+              }
+            ]
+          },
+          "description": "Whether the version is preview or stable.",
+          "readOnly": true
+        }
+      }
+    },
+    "GuardrailsAvailableVersion": {
+      "type": "object",
+      "description": "Available Guardrails Version",
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v3/types.json#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GuardrailsAvailableVersionsProperties"
+        }
+      },
+      "required": [
+        "properties"
+      ]
+    },
+    "GuardrailsAvailableVersionsList": {
+      "type": "object",
+      "description": "Hold values properties, which is array of GuardrailsVersions",
+      "properties": {
+        "value": {
+          "type": "array",
+          "x-ms-identifiers": [],
```

Comment:
>[] [](http://example.com/codeflow?start=30&length=2)

please follow [http://azure.github.io/autorest/extensions/#x-ms-identifiers](http://azure.github.io/autorest/extensions/#x-ms-identifiers) to add the identifying property

Reply:
According to http://azure.github.io/autorest/extensions/#x-ms-identifiers, if you use `id` as the identifying property, you do not need to add `x-ms-identifiers`. `[]` means there is no appropriate identifying properties.